sysutils/apl_select/Portfile

```diff
@@ -25,4 +27,4 @@ destroot {
     select::install apl ${filespath}/none
 }
 
-livecheck.type	none
+livecheck.type          none
```

Comment:
BTW, having a livecheck is a good idea in general. They're easy to do, so you should probably have one. Check another package (like gnutar say) that has a `gnu:` dist location in it.

Reply:
Thanks, done for gnu-apl.